lib/vsc/install/shared_setup.py

```diff
@@ -1564,7 +1564,9 @@ def action_target(package, *args, **kwargs):
         # but vsc-install relies on the setuptools' behaviour of ignoring failing dependency installations and
         # just continuing with the next entry in dependency_links
         'setuptools < %s' % MAX_SETUPTOOLS_VERSION,
-        'mock',
+        # mock 4.0 only supports Python 3+
+        "mock<4.0;python_version<'3.0'",
```

Comment:
put spaces around operators for rpms

Reply:
Hmm, not even sure the `;` will be digested well by the RPM stuff...

Reply:
fixed in 8ee66c8, verified with `clusterbuildrpm` using my branch (& `--queryonly`)